invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -146,6 +149,16 @@ class DatesSchema(Schema):
     description = StrippedHTML(attribute='description')
 
 
+class RightsSchema(Schema):
+    """Rights schema."""
+
+    id = fields.String()
+    title = L10NString(data_key='title_l10n')
+    description = L10NString(data_key='description_l10n')
+    link = fields.String()
+    props = fields.Dict()
```

Comment:
What is ``props``needed for?

Reply:
Because the url is inside the props and we need to display it in the landing page + deposit form
![image](https://user-images.githubusercontent.com/15194802/125782458-8282360b-b55f-4bb6-ab58-f0e9251ac6ba.png)

Reply:
I guess this ties to how much we wanna dereference and how much we wanna put in the UISerializer, no? For other vocas we are doing the later, while this seems to do the former.